config/discovery.go

```diff
@@ -57,18 +64,21 @@ type Polling struct {
 	RetryAfter Duration
 	// StopAfter overrides Discovery.PollStopAfter.
 	StopAfter Duration
+	// DeactivateAfter overrides Discovery.DeactivateAfter.
+	DeactivateAfter Duration
 }
 
 // NewDiscovery returns Discovery with values set to their defaults.
 func NewDiscovery() Discovery {
 	return Discovery{
-		LotusGateway:   "https://api.chain.love",
-		Policy:         NewPolicy(),
-		PollInterval:   Duration(24 * time.Hour),
-		PollRetryAfter: Duration(5 * time.Hour),
-		PollStopAfter:  Duration(7 * 24 * time.Hour),
-		RediscoverWait: Duration(5 * time.Minute),
-		Timeout:        Duration(2 * time.Minute),
+		LotusGateway:    "https://api.chain.love",
+		Policy:          NewPolicy(),
+		PollInterval:    Duration(24 * time.Hour),
+		PollRetryAfter:  Duration(5 * time.Hour),
+		PollStopAfter:   Duration(7 * 24 * time.Hour),
+		DeactivateAfter: Duration(7 * 24 * time.Hour),
```

Comment:
Maybe this should explicitly be set to the same value as `PollStopAfter` so that it tracks whatever that is set to:
```go
	const defaultPollStopAfter = Duration(7 * 24 * time.Hour)
...
		PollStopAfter:   defaultPollStopAfter,
		DeactivateAfter: defaultPollStopAfter,
```